schemas/software.yml

```diff
@@ -29,6 +30,7 @@ tag: "enum(
           'engineering',
           'mobile',
           'model',
+          'neuroengineering',
```

Comment:
@mirestrepo do we want to just have `neuro` over here similar to collaborations, then use an additional tag where it makes sense? so for OMNI this would be `neuro` and `engineering`

Reply:
yeah! I think that's how we settled this one on collaborations side, so I think it makes sense for software as well